src/configure.ts

```diff
@@ -21,13 +18,41 @@ export async function jestPreviewConfigure(
 
   options.externalCss?.forEach((cssFile) => {
     const basename = path.basename(cssFile);
+
     // Avoid name collision
-    // Add `global` to let `jest-preview` server that we want to cache those files
-    const destinationBasename = `cache-${basename}`;
-    const destinationFile = path.join(
-      './node_modules/.cache/jest-preview-dom',
-      destinationBasename,
-    );
+    // Example: src/common/styles.css => cache-src___common___styles.css
+    const delimiter = '___';
+    const destinationBasename = `cache-${basename.replace('/', delimiter)}`;
```

Comment:
This will replace the first `/`.
```suggestion
    const destinationBasename = `cache-${basename.replace(/\//g, delimiter)}`;
```

Reply:
I don't think this logic correct since we are using `basename`